### PR TITLE
allow loading config options from config file

### PR DESCRIPTION
**Pull request by lzurbriggen**
This is an early PoC.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,19 +34,21 @@
     "npm": ">= 3.0.0"
```
Comment (olof-nord) [inline package.json:34]: As well as specifying the NPM version, we could also add the supported Node.js version. Was there any issues with older NPM versions?

https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines

Comment (lzurbriggen) [reply package.json:34]: I think this is a remnant of the original vue-gettext, I'll try to figure out what makes sense here.

**Comment by lzurbriggen** [comment]
Closing as a pre-release version has been created. For now, the new version lives on the `next` branch.

@olof-nord Thank you for your inputs! If you want to give version 2 a shot (we didn't just introduce the config file but also replaced the whole message extraction process) please take a look at the new [docs pages](https://jshmrtn.github.io/vue3-gettext/) (note the package rename). But it's not stable yet and the docs are not quite complete.

**Comment by olof-nord** [comment]
amazing work. Seems to work very well for the axolotl vue frontend!

https://github.com/nanu-c/axolotl/pull/631

**Pull request closed without merging**